### PR TITLE
Apply black style to Bio.SeqIO

### PR DESCRIPTION
**Pull request by peterjc**
This pull request addresses issue #2008 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
@sbliven do you want to review this?

**Comment by peterjc** [comment]
As an aside, it looks like we've not been enforcing double quotes #2192 ... there's a flake8 plugin for that, might be worth using in the short term (will be redundant once we enforce black).

**Comment by peterjc** [comment]
I broke something...

```
======================================================================
ERROR: test_ambiguous_location (common_BioSQL.SwissProtUnknownPositionTest)
Loaded uniprot-xml with ambiguous location in BioSQL.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\biopython\Tests\common_BioSQL.py", line 1226, in test_ambiguous_location
    self.assertTrue(self.db.load(seqiter) == 1)
  File "C:\projects\biopython\build\lib.win32-2.7\BioSQL\BioSeqDatabase.py", line 900, in load
    for cur_record in record_iterator:
  File "C:\projects\biopython\build\lib.win32-2.7\Bio\SeqIO\__init__.py", line 689, in parse
    for r in i:
  File "C:\projects\biopython\build\lib.win32-2.7\Bio\SeqIO\UniprotIO.py", line 93, in UniprotIterator
    elem, alphabet=alphabet, return_raw_comments=return_raw_comments
  File "C:\projects\biopython\build\lib.win32-2.7\Bio\SeqIO\UniprotIO.py", line 568, in parse
    self.ParsedSeqRecord.annotations[k]
KeyError: 'created'
======================================================================
ERROR: test_ambiguous_location (common_BioSQL.SwissProtUnknownPositionTest)
```

(etc)

**Comment by peterjc** [comment]
I'm going to rebase this after #2262 is applied...

File: Bio/SeqIO/UniprotIO.py [hunk 1]
```diff
@@ -280,32 +317,53 @@ def _parse_comment(element):
                             end = int(pos_els[0].attrib["position"])
                             start = end - 1
                         else:
-                            start = int(list(loc_element.getiterator(NS + "begin"))[0].attrib["position"]) - 1
-                            end = int(list(loc_element.getiterator(NS + "end"))[0].attrib["position"])
-                    except (ValueError, KeyError):  # undefined positions or erroneously mapped
+                            start = (
+                                int(
+                                    list(loc_element.getiterator(NS + "begin"))[
+                                        0
```
[the remark this answers is not in the document]
Comment (sbliven) [reply Bio/SeqIO/UniprotIO.py]: Shortening the variable name is enough to fix the index problem. It looks like ElementTree.getiterator returns a list, so there should be little performance difference between the `list(...)[0]` and `next(iter(...))` forms. Generally I prefer the latter, but it's not a big difference format-wise.

**Comment by sbliven** [comment]
Looks good!

File: Bio/SeqIO/UniprotIO.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterjc) [reply Bio/SeqIO/UniprotIO.py]: See new commit, hopefully an acceptable compromise for this.

**Comment by peterjc** [comment]
Thanks for the careful review Spencer!

**Pull request merged**